prisma/schema.prisma

```diff
@@ -705,6 +707,8 @@ model ProjectVisibility {
   youtube                  Boolean @default(true)
   instagram                Boolean @default(true)
   xing                     Boolean @default(true)
+  mastodon                 Boolean @default(true)
+  blueSky                  Boolean @default(true) @map("blue_sky")
```

Comment:
Sorry. I would ignore it but you made this commit "[Remigrate because of typo...](https://github.com/mint-vernetzt/community-platform/pull/1095/commits/bcc89e4c7ca747041527b65c439ccf04dcefbb8b)" but blueSky is the typo 😬